src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+import duke.command.DukeCommands;
+import duke.util.*;
+
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Scanner;
+
+public class Duke {
+    /**
+     * Simple string array to store inputs
+     **/
+    private final DukeTaskList list;
+
```

Comment:
Not sure if the blank line here is necessary?

Reply:
fixed in ca26cb67916444729632edba7164a8834ff5f6e2